pl_bolts/models/self_supervised/resnets.py

```diff
@@ -287,8 +287,8 @@ def _resnet(arch, block, layers, pretrained, progress, **kwargs):
 
 @under_review()
 def resnet18(pretrained: bool = False, progress: bool = True, **kwargs):
-    r"""ResNet-18 model from
-    `"Deep Residual Learning for Image Recognition" <https://arxiv.org/pdf/1512.03385.pdf>`_
+    r"""ResNet-18 model from `"Deep Residual Learning for Image Recognition"
```

Comment:
Not sure if this will be correctly recognized as a link...

Reply:
It will now